t/data/Is_LSMB_running.html

```diff
@@ -30,7 +30,7 @@
             };
        </script>
         <script type="text/javascript" src="js/dojo/dojo.js"></script>
-        <script type="text/javascript" src="js/lsmb/main.js"></script>
+        <script type="text/javascript" src="js-src/lsmb/main.js"></script>
```

Comment:
I don't think this `js-src` change is correct. You're likely running from source locally, but travis runs built.